### PR TITLE
Tftpboot test fix

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Fixes the testsuite to also run on uyuni.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: test fixes

- [X] **DONE**

## Test coverage
- Cucumber tests were fixed

- [X] **DONE**

## Links

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/secondary/minxen_guests.feature [hunk 1]
```diff
@@ -106,7 +106,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
     Given I am on the "Virtualization" page of this "xen_server"
     When I click on "Edit" in row "test-vm"
     And I wait until I do not see "Loading..." text
-    Then I should see "512" in field "memory"
```
Comment (juliogonzalez) [inline testsuite/features/secondary/minxen_guests.feature:109]: Why is this removed?

File: testsuite/features/secondary/minxen_guests.feature [hunk 2]
```diff
@@ -117,7 +116,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
     And I enter "02:34:56:78:9a:bc" as "network0_mac"
     And I click on "Update"
     Then I should see a "Hosted Virtual Systems" text
-    And "test-vm" virtual machine on "xen_server" should have 1024MB memory and 2 vcpus
```
Comment (juliogonzalez) [inline testsuite/features/secondary/minxen_guests.feature:120]: Same

File: testsuite/features/secondary/minxen_guests.feature [hunk 3]
```diff
@@ -217,7 +214,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
     Then I should see a "Hosted Virtual Systems" text
     When I wait until I see "test-vm3" text
     And I wait at most 500 seconds until table row for "test-vm3" contains button "Stop"
-    Then "test-vm3" virtual machine on "xen_server" should have 512MB memory and 1 vcpus
```
Comment (juliogonzalez) [inline testsuite/features/secondary/minxen_guests.feature:220]: And same

**Review by juliogonzalez — COMMENTED** [review]
Since you made me a reviewer, let's merge it even if I have questions.

That way we'll have results for Monday.

But please have a look on Monday and clarify, and address any other reviewers' requests.

Thanks!

File: testsuite/features/secondary/minxen_guests.feature (hunk 1 above)
Comment (cbosdo) [reply testsuite/features/secondary/minxen_guests.feature:109]: This was detailed in the commit description: on Xen there may be a tiny difference between the memory we want to set and the one the VM actually had. Even if there usually isn't such a difference, this may happen and thus falsely break the tests.

File: testsuite/features/secondary/minxen_guests.feature (hunk 2 above)
Comment (cbosdo) [reply testsuite/features/secondary/minxen_guests.feature:120]: same story with Xen and memory

File: testsuite/features/secondary/minxen_guests.feature (hunk 3 above)
Comment (cbosdo) [reply testsuite/features/secondary/minxen_guests.feature:220]: and here again

File: testsuite/features/secondary/minxen_guests.feature (hunk 1 above)
Comment (juliogonzalez) [reply testsuite/features/secondary/minxen_guests.feature:109]: Didn't notice. For such small PRs I don't review the commit history.

Anyway, explained now, and it makes sense. Resolving.
